### DISCUSSION TITLE
V0.7 virus detected

### DISCUSSION THREAD

**Discussion by f4axk** [egzumer/uvk5-chirp-driver#17] [Q&A]
CHIRP_egzumer_V0.7 virus detected

![Capture d’écran 2024-01-02 103429](https://github.com/egzumer/uvk5-chirp-driver/assets/68162981/e52a7a8f-625c-40dd-8066-658c780ce3cd)

![Capture d’écran 2024-01-02 104212](https://github.com/egzumer/uvk5-chirp-driver/assets/68162981/40e3aa91-23e1-4db6-9dfc-a13a270d07a7)

**Comment by egzumer** [comment]
There is no virus, there is a note in the readme that you didn't read
